### DISCUSSION TITLE
app doesnt uoload from multiple phones

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9593] [Q&A]
### The bug

1. Install app on iphone and start uploading whats app videos
2. After few days on your 2nd phone, install app on android phone n set whats app videos to be uploaded

Issue. Iphone uploads doesnt work anymore

### The OS that Immich Server is running on

win 10

### Version of Immich Server

latest

### Version of Immich Mobile App

latest

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Default
```

### Your .env content

```Shell
Default
```

### Reproduction steps

```bash
1.see above
2.
3.
...
```

**Comment by bo0tzz** [comment]
There is no interaction between installations of the app on different devices. More likely, this is just Apple/iOS not cooperating with the background upload process.

**Comment by amgschnappi** [comment]
![IMG_1443](https://github.com/immich-app/immich/assets/102602534/b8bca561-6102-4c46-909e-2f9cdc0022ec)

no. It says nothing to upload
